docker-compose.yml

```diff
@@ -34,7 +34,9 @@ services:
     networks:
       - back-net
     environment:
+      - POSTGRES_USER=postgres
       - POSTGRES_PASSWORD=testing
+      - POSTGRES_DB=db_banking
```

Comment:
I think this method does not work, I tried in my local with a new Postgres container and got this following error.
```
Starting banking system service...
docker-compose up -d
django-simple-banking-api_bank_db_1 is up-to-date
django-simple-banking-api_reverse_proxy_1 is up-to-date
django-simple-banking-api_bank_api_1 is up-to-date
Creating the bank database and performing migrations...
docker-compose exec bank_api python manage.py migrate
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/db/backends/base/base.py", line 220, in ensure_connection
    self.connect()
  File "/usr/local/lib/python3.8/site-packages/django/utils/asyncio.py", line 26, in inner
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/django/db/backends/base/base.py", line 197, in connect
    self.connection = self.get_new_connection(conn_params)
  File "/usr/local/lib/python3.8/site-packages/django/utils/asyncio.py", line 26, in inner
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/django/db/backends/postgresql/base.py", line 185, in get_new_connection
    connection = Database.connect(**conn_params)
  File "/usr/local/lib/python3.8/site-packages/psycopg2/__init__.py", line 127, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: FATAL:  database "db_banking" does not exist


The above exception was the direct cause of the following exception:
```